corporate/views/billing_page.py

```diff
@@ -123,7 +124,11 @@ def remote_realm_billing_page(
     if customer is None or get_current_plan_by_customer(customer) is None:
         return HttpResponseRedirect(reverse("remote_realm_upgrade_page", args=(realm_uuid,)))
 
-    main_context = billing_session.get_billing_page_context()
+    try:
+        main_context = billing_session.get_billing_page_context()
+    except MissingDataError:
+        return render(request, "corporate/server_not_uploading_data.html")
```

Comment:
So this case should be quite rare, in that it's hard to log in to this flow for `RemoteRealmSession` without data being uploading (i.e. the most likely failure modes will be the server being down or a firewall, and both would prevent using the login flow). I think a generic error page like this should be OK for that -- and their action should be to have their IT person look at logs and contact support.

The `RemoteZulipServer` case, at least if the version is less than 8.0, has the following as common reasons:
* The server is on any version and a firewall prevents connecting to the service. (This should also mean push notifications don't work).
* The server is pre-8.0 and `SUBMIT_USAGE_STATISTICS=False`. This is the case where they need to ask their sysadmin to do something, but the options are fairly specific: Either go upgrade your server to 8.0 or enable that setting.

Reply:
OK https://github.com/zulip/zulip/pull/28105 will be responsible for answering the "Is it 8.0 or newer" question.

Reply:
Updated to pass two context variables to the template in each of the call sites:
- `remote_realm_session` boolean: set to false for remote Zulip server billing sessions
- `last_api_featurel_level` None or int: set this as None for the remote realm cases

Didn't add a billing session method for adding these context variables as it's not applicable for the non-remote case, but we could do so.